osmdroid-android/src/main/java/org/osmdroid/util/BoundingBox.java

```diff
@@ -66,13 +66,28 @@ public BoundingBox concat(BoundingBox bb2){
 	// ===========================================================
 
 	/**
+	 * Use {@link #getCenterWithDateLine()} instead to take date line into consideration
 	 * @return GeoPoint center of this BoundingBox
 	 */
+	@Deprecated
 	public GeoPoint getCenter() {
 		return new GeoPoint((this.mLatNorth + this.mLatSouth) / 2.0,
 				(this.mLonEast + this.mLonWest) / 2.0);
 	}
 
+	/**
+	 * This version takes into consideration the date line
+	 * @since 5.6.6
+	 */
+	public GeoPoint getCenterWithDateLine() {
```

Comment:
why not just put this code in get center?

Reply:
I didn't want to break anything. I don't know who uses `getCenter`: maybe people now admit that `east` and `west` in `BoundingBox` only mean `longitude1` and `longitude2`, and populate them without any relevant order.